sources/Algo/include/Algo.h

```diff
@@ -133,16 +134,27 @@ struct GeneratorDefinition {
     WITH_IMPEDANCE_DIAGRAM_PQ_SIGNALN  ///< Use GeneratorPVWithImpedanceDiagramPQSignalN
   };
 
+  using ReactiveCurvePoint = DYN::GeneratorInterface::ReactiveCurvePoint;  ///< Alias for reactive curve point
+
   /**
    * @brief Constructor
    *
    * @param genId generator id
    * @param type the model to use
+   * @param nodeid the node id connected to the generator
```

Comment:
typo: nodeId

Reply:
No, in the parameters, it is nodeid

Reply:
Please respect the coding style independently from the parameters

Reply:
OK